cmd/otelcontribcol/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
+
+FROM scratch
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY otelcontribcol /usr/bin/
```

Comment:
nit: we put OTelCol straight on the root, we should follow a consistent pattern between these two. it can be either one but it should be consistent.